Client/Frontend/Browser/BrowserPrompts.swift

```diff
@@ -35,12 +42,33 @@ class JSPromptAlertController: UIAlertController {
 protocol JSAlertInfo {
     func alertController() -> JSPromptAlertController
     func cancel()
+    /// Informs JSPromptAlertController that the completion handler has already been called.
+    var didComplete: Bool { get }
```

Comment:
This variable is not very clearly named. Something like `completionHandlerAlreadyCalled` would be more clear as it appears through the rest of the code.

Reply:
Fixed